apps/demo/styles/globals.css

```diff
@@ -1,3 +1,26 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+/* Example on overriding packages/js colors */
+.dark {
+  --fb-primary: rgb(0, 157, 168);
+  --fb-on-primary: black;
+  --fb-border: var(--slate-300);
+  --fb-border-highlight: var(--slate-500);
+  --fb-ring-focus: var(--slate-500);
+  --fb-text: #f1ffff;
+  --fb-text-2: rgb(226, 232, 240);
+  --fb-text-3: rgb(209, 209, 209);
+  --fb-text-4: rgb(165 164 170);
+  --fb-bg: #080b15;
+  --fb-bg-2: rgb(44, 50, 60);
+  --fb-bg-selected: rgb(30, 34, 41);
+  --fb-placeholder: rgb(168, 168, 168);
+  --fb-tyc-shadow: rgb(190, 190, 190);
+  --fb-rating-fill: var(--yellow-300);
+  --fb-rating-hover: var(--yellow-500);
+  --fb-back-btn-border: currentColor;
+  --fb-submit-btn-border: transparent;
+  --fb-rating-selected: black;
```

Comment:
@kymppi Thank you so much for your contribution 🎉 . The contribution looks great and can be an awesome starting point for us in the direction of offering the customizability option.

I would request you to change these variable names to a bit more intuitive ones so that it gets easier for other contributors to understand what each variable targets.

Example:
- The `--fb-primary: #383ffc;` variable targets the button color in the survey, we could name it something like, `--fb-button-background: #383ffc;`, or something similar. The idea is the the contributors get the idea from the name itself of the element/area they are targeting on the UI to make things easier.

Once you make these changes, I'll re-review this.

Reply:
Yep! I need to rename the variables.
But in fact that primary color is not only for the button. It's also used for the progress bar.